corehq/apps/userreports/reports/filters/choice_providers.py

```diff
@@ -84,7 +84,7 @@ def get_choices_for_values(self, values, user):
         used_values = {value for value, _ in choices}
         for value in values:
             if value not in used_values:
-                choices.add(Choice(value, unicode(value) if value is not None else ''))
+                choices.add(Choice(value, str(value) if value is not None else ''))
```

Comment:
shouldn't this be six.text_type? or does six fix that?

Reply:
On closer look this looks wrong to me, not sure why modernize would have done that.